Dockerfile

```diff
@@ -28,6 +28,8 @@ RUN asdf install
 ENV IPFS_PATH=/asdf/.ipfs
 RUN mkdir -p ${IPFS_PATH} && echo "/ip4/127.0.0.1/tcp/5001" > ${IPFS_PATH}/api
 
+RUN go install github.com/guseggert/glibc-check/cmd/glibc-check@v0.0.0-20211020180227-964e8f06bf27 && asdf reshim
```

Comment:
This will work only on CI and `./dockerized` builds.

Can we move this to the end of `./deps-check.sh`  so the build works outside of docker?
(An alternative is to forbid non-dockerized builds by adding a check to ./deps-check.sh` informing user that build can only be run  via `./dockerized` wrapper, but I think it's better to keep the build working everywhere, makes dev and debug easier).

Reply:
Ah yeah fair enough, how about I add a check to `./deps-check.sh`, it would be surprising for that to install stuff...can just leave that up to the dev.

Reply:
yes, sgtm